design/one-pager-strongly-typed-class.md

```diff
@@ -0,0 +1,156 @@
+# Strongly Typed Resource Classes
+* Owner: Daniel Mangum (@hasheddan)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Terminology
+
+* **Abstract Resource**: a resource type that a `ResourceClaim` can specify as its kind. They are general and portable across managed service providers (i.e. `MySQLInstance`).
+* **Resource Class**: a Kubernetes resource that contains implementation details specific to a certain environment or deployment, and policies related to a kind of resource.
+* **Resource Claim**: a Kubernetes resource that captures the desired configuration of a resource from the perspective of a workload or application.
+
+## Background
+
+Crossplane provisions resources by allowing resource claims to specify a reference to a resource class, or fall back on a [default resource class](one-pager-default-resource-class.md) for the claim's kind. The `kind` of a resource claim is a portable infrastructure type that may have concrete implementations on multiple providers (i.e. `MySQLInstance`, `Bucket`, `RedisCluster`, etc.). Resource classes are all of `kind: ResourceClass` and specify a provisioner that dictates the actual provider implementation. For example, the following resource class is of `kind; ResourceClass`, but specifies its `provisioner` as `rdsinstance.database.aws.crossplane.io/v1alpha1`. 
```

Comment:
I agree, especially considering the preceding sentence. I think if the "but" was just changed to an "and" then it would communicate the correct sentiment.